classes/Configuration/Configuration.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\PrestashopCheckout\Configuration;
+
+use Module;
+use Psr\Log\LogLevel;
+
+class Configuration
```

Comment:
If you're still to rename this class so there's no misunderstanding with the core class, there is something like `RuntimeEnvironment` which could be a fit

Reply:
I need data from .env, constants, PrestaShop Configuration, PrestaShop Context, Module...
I will try somethings with [symfony/config](https://symfony.com/doc/current/components/config.html) and [symfony/dotenv](https://symfony.com/doc/current/components/dotenv.html)